python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -639,28 +488,27 @@ def get_params(self, deep=True):
         -----------
         deep : boolean (default = True)
         """
-        params = dict()
-        for key in RandomForestClassifier.variables:
-            var_value = getattr(self, key, None)
-            params[key] = var_value
-        return params
+        return self._get_params(deep)
 
-    def set_params(self, **params):
+    def set_params(self, worker_numb=None, **params):
         """
         Sets the value of parameters required to
         configure this estimator, it functions similar to
         the sklearn set_params.
 
         Parameters
         -----------
-        params : dict of new params
+        params : dict of new params.
+        worker_numb : list (default = None)
+            If worker_numb is `None`, then the parameters will be set for all
```

Comment:
Are there cases where a user would want only a subset of workers to set different parameters?

Reply:
For example if the user wants to change the number of trees in one model. The way it is now the user can change it just for one model.
The idea behind this approach was to provide the user with flexibility while changing the parameters.